### PR TITLE
Link Exclusive Accordion to web-features id

### PR DESCRIPTION
**Pull request by foolip**
This is using the same tagging approach as in BCD:
https://github.com/mdn/browser-compat-data/blob/74547d66328d241c6b7d171adc3cdf7bce8872dd/html/elements/dialog.json#L9

### CODE REVIEW

**Comment by foolip** [comment]
@SachaG would you prefer a new `webFeatures` field?

File: entities/features/html_features/features_html.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@
       title: Chrome Platform Status
   tags:
     - features
+    - web-features:details-name
```
Comment (SachaG) [inline entities/features/html_features/features_html.yml]: I think maybe it should be its own separate field? It's a bit weird to use the tags field for that?

**Comment by SachaG** [comment]
> @SachaG would you prefer a new `webFeatures` field?

Yes that would make more sense to me. In fact I think ideally we would just use the web feature id as the canonical `id` field, but that might not be practical for backwards-compatibility reasons.

**Comment by SachaG** [comment]
Maybe we could name the field something like `webFeaturesId`, or `wfId`? `web-features` by itself doesn't really communicate the fact that it's a canonical ID?

**Comment by foolip** [comment]
@SachaG `webFeaturesId` works for me, it's pretty explicit and easy to search for. I'll update this PR to show what it'd look like.

**Comment by SachaG** [comment]
1 done, 678 to go! (I'm guessing, I didn't count…)

**Pull request merged**